prototypes/styletransfer/model.py

```diff
@@ -191,6 +192,26 @@ def _estimate_content2weight(
 
         return content / style
 
+    def run_style_transfer_video(
+        self,
+        frames: List[np.array],
+        style_img: np.array,
+        num_iterations=100,
+        content_weight=1.0,
+        style_weight=1.0,
+        total_variation_weight=1.0
+    ) -> List[np.array]:
+
+        nframes = len(frames)
+        transferred = []
+        for frame in range(0, nframes-1):
+            content_img = frame
+            for result in self.run_style_transfer(style_img=style_img, content_img=content_img, num_iterations=num_iterations): 
```

Comment:
maybe we should pass the weights here?

Reply:
@gabrielelanaro let me know if 4ad290c fixes this

Reply:
I also added the unit test, but still need to actually run it. The test regarding `extract_frames_from_gif` passes.